lib/tpm2_session.c

```diff
@@ -243,9 +239,9 @@ tool_rc tpm2_session_restore(ESYS_CONTEXT *ctx, const char *path, bool is_final,
 
     FILE *f = fopen(dup_path, "rb");
     if (!f) {
-        LOG_ERR("Could not open path \"%s\", due to error: \"%s\"",
-                dup_path, strerror(errno));
         free(dup_path);
+        LOG_ERR("Could not open path \"%s\", due to error: \"%s\"", dup_path,
+                strerror(errno));
```

Comment:
When you re-based you got the order of this hunk wrong, the free must go after the LOG_ERR statement. Please go to that commit and edit it so it's correct.

Reply:
Thanks!